schemas/groups/electrical.json

```diff
@@ -177,32 +177,73 @@
       }
     },
 
-    "acSource": {
+    "acQuantities": {
       "type": "object",
-      "title": "AC Source",
-      "description": "AC power source common values",
+      "title": "AC Quantities",
+      "description": "AC equipment common quantities",
       "properties": {
         "associatedBus": {
           "type": "string",
           "description": "Name of BUS source is assocated with (if applicable, may = NULL)"
         },
-
-        "voltage": {
-          "type": "object",
-          "properties": {
-            "measured": {
-              "description": "Measured voltage at AC Source terminals",
-              "$ref": "../definitions.json#/definitions/numberValue",
-              "units": "V"
-            }
-          }
+        "lineNeutralVoltage": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "RMS voltage measured between phase and neutral.",
+          "units": "V"
+        },
+        "lineLineVoltage": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "RMS voltage measured between phases",
+          "units": "V"
+        },
+        "current": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "RMS current",
+          "units": "A"
+        },
+        "frequency": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "AC frequency.",
+          "units": "Hz"
+        },
+        "reactivePower": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "Reactive power",
+          "units": "W"
+        },
+        "powerFactor": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "Power factor",
+          "unit": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX fraction"
```

Comment:
We have `ratio` with description 'Relative value compared to reference or normal value. 0 = 0%, 1 = 100%, 1e-3 = 1 ppt' which fits perfectly for this.